src/transport/SecureSessionTable.h

```diff
@@ -142,9 +142,13 @@ class SecureSessionTable
         }
 
         const Transport::SecureSession * operator->() const { return mSession; }
+        auto GetNumMatchingOnFabric() { return mNumMatchingOnFabric; }
+        auto GetNumMatchingOnPeer() { return mNumMatchingOnPeer; }
 
     private:
         SecureSession * mSession;
+        uint16_t mNumMatchingOnFabric;
```

Comment:
We need to static_assert that the pool size is not large enough to allow these to overflow.